main.go

```diff
@@ -174,6 +174,40 @@ func main() {
 			Before:      beforeConnectAction,
 			Action:      connectAction,
 		},
+		{
+			Name:      "configure",
+			Usage:     "Configure the host",
```

Comment:
Looking at this text surrounded by the other text in the help output, I think
this needs to change to something like:

"Configures the behavior of the system"

```
COMMANDS:
   connect     Connects the system to Red Hat
   configure   Configure the host
   disconnect  Disconnects the system from Red Hat
   status      Prints status of the system's connection to Red Hat
   help, h     Shows a list of commands or help for one command
```